希望@douyinfe/semi-icons-lab可以支持react更高版本

npm i @douyinfe/semi-icons-lab
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: witcube-frontend@0.1.0   
npm ERR! Found: react@19.0.0
npm ERR! node_modules/react
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0 || ^17.0.0 || ^18.0.0" from @douyinfe/semi-icons-lab@2.75.0
npm ERR! node_modules/@douyinfe/semi-icons-lab
npm ERR!   @douyinfe/semi-icons-lab@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! C:\Program Files\nodejs\node_cache\_logs\2025-02-27T07_06_33_305Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: C:\Program Files\nodejs\node_cache\_logs\2025-02-27T07_06_33_305Z-debug-0.log

--- a/packages/semi-icons-lab/package.json
+++ b/packages/semi-icons-lab/package.json
@@ -50,8 +50,8 @@
     "ts-loader": "^5.4.5"
   },
   "peerDependencies": {
-    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
-    "react-dom": "^16.8.0 || ^17.0.0 || ^18.0.0"
+    "react": ">=16.0.0",
+    "react-dom": ">=16.0.0"
   },
   "_unpkg": true,
   "unpkgFiles": [

--- a/packages/semi-icons/package.json
+++ b/packages/semi-icons/package.json
@@ -50,7 +50,7 @@
         "ts-loader": "^5.4.5"
     },
     "peerDependencies": {
-        "react": "^16.8.0 || ^17.0.0 || ^18.0.0"
+        "react": ">=16.0.0"
     },
     "_unpkg": true,
     "unpkgFiles": [

--- a/packages/semi-illustrations/package.json
+++ b/packages/semi-illustrations/package.json
@@ -22,7 +22,7 @@
         "url": "https://github.com/DouyinFE/semi-design"
     },
     "peerDependencies": {
-        "react": "^16.8.0 || ^17.0.0 || ^18.0.0"
+        "react": ">=16.0.0"
     },
     "devDependencies": {
         "case-sensitive-paths-webpack-plugin": "2.4.0",

